2020/13xxx/CVE-2020-13671.json

```diff
@@ -1,18 +1,87 @@
 {
```

Comment:
{} is not allowed for {'problemtype_data': [{'description': [{'lang': 'eng', 'value': 'Remote code execution'}]}]}